data/json/dreams.json

```diff
@@ -376,6 +391,7 @@
   {
     "type": "dream",
     "messages": [
+      "You have a nightmare of twisting, endless corridors⋯",
```

Comment:
Still feels pretty far from cattle

Reply:
Yeah I didn't write this one (it came from user on reddit) so I wanted to preserve it if possible.  I figured this way it could be either a minotaur or a slaughterhouse stockade?

Ok for me to chat you on discord to discuss the remaining open conversations? Most of them I've closed simply by deleting the offending content, or incorporating feedback, but Chimera, Insect, and Mouse are still open.

Reply:
TBF all of the cattle dreams are problematic.  It needs to be about things cattle DO, not things that are done to cattle.